EdFi.Buzz.Etl/processors/surveyProcessor.js

```diff
@@ -33,13 +34,10 @@ async function getDB() {
   return client;
 }
 
-async function isSurveyLoader(staffkey, db) {
+async function isAdminSurveyLoader(staffkey, db) {
```

Comment:
The deleted changes will remove the ability for teachers to upload surveys. I recommend not replacing this code.